six/modules/c++/six.sicd/source/ComplexXMLParser04x.cpp

```diff
@@ -337,7 +337,7 @@ XMLElem ComplexXMLParser04x::convertAntennaParamArrayToXML(
     //! optional field in 0.4
     if (array)
     {
-        XMLElem arrXML = newElement("Array", apXML);
+        XMLElem arrXML = newElement(name, apXML);
```

Comment:
This is not right... if you look up a level to `ComplexXMLParser::convertAntennaToXML()`, the name that is sent down is `Tx`, `Rcv`, or `TwoWay`, not `Array`.  It looks like the reason `name` is sent down is so that `ComplexXMLParser10x::convertAntennaParamArrayToXML()` can throw and include the name in there... `ComplexXMLParser04x` doesn't need to do that (that field must be optional), so instead you want to just remove the name of the variable in `ComplexXMLParser04x.cpp`